---
title: "Bridging levels - the importance of cell type identity"
draft: true
author: "Lukas Anneser"
date: 2025-05-17
categories: [neuroscience, journal club, analysis]
---
 
## From single neuron recordings to population level analysis
Over recent decades, one major theme of neuroscience was a switch from the analysis of individual neurons to recording ever-larger numbers of cells and trying to understand their joint behavior - the population activity. In many ways, this development was unavoidable and the result of a simple numbers game: while some invertebrate species only have a small number of highly stereotyped neurons (in the nematode C. elegans, all 302 neurons are even uniquely named and can be identified across animals), most animals have far over 100 000 neurons. As one enters the class of mammals, that number can safely be assumed to be in the millions. 

In order to better understand what the concurrent activity in all these cells mean, it is not sufficient to simply increase the number of neurons one records from. In the end, one is left with a gigantic matrix to make sense of and without a way to simplify, that is a rather daunting task . One important insight was that in a population of neurons, the activity of some subpopulations tends to be correlated. Depending on the strength of correlations, dimensionality reduction techniques such as principal component analysis can be applied, allowing for a lower-dimensional representation of the dataset without sacrificing too much variability. Treating neuronal population recordings this way allowed for fascinating discoveries, showcasing how the topology of the subspace carved out by neuronal dynamics is intricately linked to the function of the circuit. 

Of course, there is no free lunch and [data processing can only ever destroy information](https://en.wikipedia.org/wiki/Data_processing_inequality). What is sacrificed in this context is knowledge about individual cells and their responses. Now, one might assume that the trade-off is worth it, but progress in biology is dependent on the combination of insights on [different levels of understanding](https://global.oup.com/academic/product/explaining-the-brain-9780199299317?cc=de&lang=en&). I recently came across a really interesting paper from the lab of [Liset Menendez de la Prida](https://cajal.csic.es/en/neural-circuits/), that is beautifully illustrating this point. The publication is called [*Cell-type-specific manifold analysis discloses independent geometric transformations in the hippocampal spatial code*](https://doi.org/10.1016/j.neuron.2025.01.022) and is quite a read. Julio Esparza and colleagues try to understand the role of genetically defined subpopulations of neurons in the hippocampal subregion CA1 in the formation of spatial representations. 

## Hippocampal representation of a linear track 
The ability of the hippocampal formation to represent the place of an animal within its spatial context has been explored for [over 50 years](https://www.sciencedirect.com/science/article/abs/pii/0006899371903581). Most notably, when animals move through an environment, some hippocampal cells will only become active whenever a certain place is visited. These kind of cells became known as **place cells** and form one of the bedrocks of the neuroscientific field of spatial navigation. 

The cell type identity of these neurons is almost never explicitly addressed. In early electrophysiological recordings, it was simply not possible to discern what kind of neuron one recorded from, and even when it became feasible, questions along these lines never took center-stage. However, the hippocampus is not a monolithic tissue with only one kind of neuron, but an intricate anatomical structure made up of a [diverse set of cell types](https://www.science.org/doi/full/10.1126/science.aaa1934). This is indeed a general feature of neural circuits that, with the advent of single-cell RNA-sequencing, has become more and more appreciated and to refer to the excitable cells inside the brain simply as *neurons* doesn`t do justice to the multiplicity of shapes and functions observed and described. In the paper, Esparza et al. ask about the contribution of differently located pyramidal cells (PCs) to the representation of a linear track. To be precise, they are able to perform targeted recordings either from superficial or deep PCs (they do this be using either transgenic animals or deliver calcium reporters to gene populations of interest via virus injections). 

This approach facilitates a peculiar take on the population level analysis. Normally, one would temporally bin neuronal activity and depict the firing rate (or the corresponding proxy) of each neuron on one axis of an n-dimensional space, with n being the number of recorded neurons. In this space, one can now analyse the features of the subspace being carved out by the population analysis. In this case, however, the authors can additionally ask what kind of properties are contributed by which cell types. For a first characterization, they simply captured hippocampal activity from an animal running back and forth on a linear track. Consistent with what one would expect from previous research, the resulting subspace occupied by neuronal activity (the **manifold**) is best described as a tridimensional ring for both kind of neuronal subpopulations. However, the representations differed in subtle ways the authors took great care to characterize. As a side note, the methodological considerations put forward were one of the highlights of this publication for me. Every step of the analysis was motivated with great care and described simply and intuitively. I would recommend to check out the [github repo](https://github.com/PridaLab/hippocampal_manifolds) to get an impression of what accompanying code should look like. For example, the authors use a method called *angle-based intrinsic dimensionality* (ABID) as described by [Thordsen et al., 2020](https://doi.org/10.1007/978-3-030-60936-8_17) to determine the above mentioned tridimensionality of the observed manifold. This method computes the directional vectors for points neighboring a reference point in an n-dimensional space, and computes the average pairwise cosine similarity for these vectors. The intrinsic dimensionality is then given by the inverse of this value. If the data lives in a high-dimensional space, the vectors will point in random directions and the cosine similarity is low, thus resulting in a high ABID value. Pure algorithmic beauty! Anyway, back to the paper. 

## Changing spatial cues, changing manifolds

The most striking observation about the ring-like manifolds was the difference in their responses to the manipulation of spatial cues. As a simple experimental intervention, the track was simply *rotated* inside the lab, resulting in a 180 ° mismatch between local and global cues for the rodents navigating the track before and after this rotation. For both superficial and deep neurons, the manipulation resulted in a displacement of the ring. However, the ring-like representation of the deep neurons was *additionally* rotated by 180 °, matching the change in the outside world.  

